scripts/hail_batch/README.md

```diff
@@ -0,0 +1,10 @@
+# Generate PCA loadings, eigenvalues, and scores
+This runs a Hail query script in Dataproc using Hail Batch in order to output the PCA eigenvalues, scores, and loadings from the 1KG + HGDP dataset. To run, use conda to install the analysis-runner, then execute the following command:
+
+```
+analysis-runner --dataset ancestry \
+--access-level test --output-dir “gs://cpg-ancestry-main/Kat_PCA_loadings" \
```

Comment:
You'll need `full` access in order to _write_ to the `main` bucket. Also how about versioning this data, just in case?

```suggestion
--access-level full --output-dir “gs://cpg-ancestry-main/1kg_hgdp_pca/v0" \
```

Reply:
versioning sounds 👌